EpsilonLights/Src/Lights.c

```diff
@@ -58,6 +58,15 @@ void updateLightsTask(void const* arg)
             HAL_GPIO_WritePin(BRAKE_GPIO_Port, BRAKE_Pin, LIGHT_OFF);
         }
 
+        if (regenInputs[REGENBRAKE_INPUT_INDEX_P1] == 0 && regenInputs[REGENBRAKE_INPUT_INDEX_P2] != 0)
```

Comment:
Combine this with the previous loop, right now the mechanical brake will be ignored because of this statement